addons/l10n_ca/views/report_template.xml

```diff
@@ -1,31 +1,55 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <template id="l10n_ca_external_layout_standard" inherit_id="web.external_layout_standard">
-        <xpath expr="//div[hasclass('row')][last()]" position="after">
+        <xpath expr="//ul[@name='company_address_list']" position="inside">
             <t t-if="company.account_fiscal_country_id.code == 'CA' and company.l10n_ca_pst" class="mt16">
-                <div>PST: <span t-field="company.l10n_ca_pst"/></div>
+                <li>PST: <span t-field="company.l10n_ca_pst"/></li>
             </t>
         </xpath>
     </template>
```

Comment:
It might be a good opportunity to have one template:
```
<template id="foo">
    <t t-if="company.account_fiscal_country_id.code == 'CA' and company.l10n_ca_pst">
        <li>PST: <span t-field="company.l10n_ca_pst"/></li>
    </t>
</template>
```
And then, for each of the external layouts:
```
<template id="l10n_ca_external_layout_standard" inherit_id="web.external_layout_standard">
    <xpath expr="//ul[@name='company_address_list']" position="inside">
        <t t-call="l10n_ca.tmp42"/>
    </xpath>
</template>
```